### PR TITLE
Add 403 return errors to URIs dealing with revisions

### PR DESCRIPTION
**Pull request by d00rman**
Forgot to address that in #180 , so adding it separately.

### CODE REVIEW

File: specs/mediawiki/v1/content.yaml [hunk 1]
```diff
@@ -558,6 +582,28 @@ definitions:
         type: string
       uri:
         type: string
+  accessDenied:
```
Comment (gwicke) [inline specs/mediawiki/v1/content.yaml]: This is a https://tools.ietf.org/html/draft-nottingham-http-problem

File: specs/mediawiki/v1/content.yaml [hunk 2]
```diff
@@ -558,6 +582,28 @@ definitions:
...
+    required:
+      - code
```
Comment (gwicke) [inline specs/mediawiki/v1/content.yaml]: The code is in the http status.

File: specs/mediawiki/v1/content.yaml [hunk 3]
```diff
@@ -558,6 +582,28 @@ definitions:
...
+      - method
```
Comment (gwicke) [inline specs/mediawiki/v1/content.yaml]: I believe this is in a req sub-object.

File: specs/mediawiki/v1/content.yaml [hunk 4]
```diff
@@ -558,6 +582,28 @@ definitions:
...
+      code:
+        type: integer
+        format: int32
+      method:
+        type: string
+      title:
+        type: string
+      type:
+        type: string
+      description:
```
Comment (gwicke) [inline specs/mediawiki/v1/content.yaml]: s/description/detail/

File: specs/mediawiki/v1/content.yaml [hunk 5]
```diff
@@ -558,6 +582,28 @@ definitions:
...
+        type: string
+      uri:
```
Comment (gwicke) [inline specs/mediawiki/v1/content.yaml]: req.uri?

Comment (d00rman) [inline specs/mediawiki/v1/content.yaml]: ups, c/p error :)

**Comment by d00rman** [comment]
@gwicke I have changed `accessDenied` to `httpError` (and updated the fields accordingly). Should we make it as the default schema for all errors, since we have field whitelisting allowing only certain error properties to see the light of day?

File: specs/mediawiki/v1/content.yaml [hunk 6]
```diff
@@ -558,6 +582,21 @@ definitions:
         type: string
       uri:
         type: string
+  httpError:
+    required:
+      - method
```
Comment (gwicke) [inline specs/mediawiki/v1/content.yaml]: I wonder if we should move 'method' down into a req sub-object that has just the uri & method.

**Pull request merged**